src/initializers/koa/parse-querystring.ts

```diff
@@ -0,0 +1,9 @@
+import { Context, Next } from 'koa';
+import * as qs from 'qs';
+
+export default async function parseQuerystring(ctx: Context, next: Next) {
+  if (ctx.querystring) {
+    ctx.state.query = qs.parse(ctx.querystring);
```

Comment:
I am not sure how performant qs.parse is. But I would add it optionally. E.g. only when someone accesses ctx.state.query

Reply:
How could you know that someone is accessing `ctx.state.query`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty